File: tasks/lectures/handong/c/10.21/1.c
#include <stdio.h>

int Morgan(int a, int b) { return !(!a*!b); }

int main() {
    int a, b;
    printf("input bool data: ");
    scanf("%d %d", &a, &b);
    printf("OR 연산 출력: ");
    printf(Morgan(a, b) ? "TRUE" : "FALSE");
    return 0;
}

File: tasks/lectures/ssu/python/4th-week/1.py
n = int(input())
for i in [2, 3, 5]:
    if n % i == 0:
        print('{}은 {}의 배수입니다.'.format(n, i))
    else:
        print('{}은 {}의 배수가 아닙니다.'.format(n, i))